com.sap.cloud.lm.sl.cf.persistence/src/main/java/com/sap/cloud/lm/sl/cf/persistence/services/ProcessLogsPersistenceService.java

```diff
@@ -50,66 +48,49 @@ public ProcessLogsPersistenceService(DataSourceWithDialect dataSourceWithDialect
 
     public String getLogContent(String space, String namespace, String logName) throws FileStorageException {
         final StringBuilder builder = new StringBuilder();
-        String logId = findFileId(space, namespace, logName);
+        List<String> logIds = findAndSortFileIds(space, namespace, logName);
```

Comment:
Either replace the method with the lambda or rename it to something like getFileIds